core/translate.c

```diff
@@ -82,7 +82,7 @@ typedef struct _translate_walk_t {
     /* PR 263407: Track registers spilled since the last cti, for
      * restoring indirect branch and rip-rel spills
      */
-    bool reg_spilled[REG_SPILL_NUM];
+    uint reg_spilled[REG_SPILL_NUM];
```

Comment:
Explain in the comment above what this uint encodes

Reply:
Done.